inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/convert.cpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "single_layer_tests/convert.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+
+std::vector<std::vector<std::vector<size_t>>> inShapes = {
+        {{1}},
```

Comment:
As for me looks like extra cases. For convert layer shapes are not really important.

Reply:
Agree, TBD

Reply:
Done